order/supplier/receive/simple-order-event.md

```diff
@@ -0,0 +1,249 @@
+---
+description: How to receive a simple order sent by the buyer.
+---
+
+# Receive an order
+
+Tradecloud will send a purchase order, either new or updated, to the supplier when an order event has been triggered.
```

Comment:
Only if the supplier is integrated with the webhook, right? I think this can use some elaboration.

Reply:
Improved

Reply:
I don't see the change?

Reply:
See the additional lines below and https://app.gitbook.com/o/-LNyIUZSvpZWZ81yEr7V/s/-LNyJ9UuwLCjTvA2sqQR-887967055/~/revisions/nZCqdUTDMvvrpj07rcDN/processes/order/supplier/receive/simple-order-event

![image](https://github.com/tradecloud/tradecloud-docs-api-v2/assets/1092388/827d9b36-9281-42f6-a82f-9d75855f1653)